.mvn/extensions.xml

```diff
@@ -2,6 +2,6 @@
   <extension>
     <groupId>io.jenkins.tools.incrementals</groupId>
     <artifactId>git-changelist-maven-extension</artifactId>
-    <version>1.0-beta-3</version>
+    <version>1.0-beta-7</version>
```

Comment:
Required to avoid
```
[WARNING] Rule 1: io.jenkins.tools.incrementals.enforcer.RequireExtensionVersion failed with message:
Detected git-changelist-maven-extension Version: 1.0-beta-3 is not in the allowed range [1.0-beta-4,).
```